WireFoundation/Package.swift

```diff
@@ -4,26 +4,34 @@
 import PackageDescription
 
 let package = Package(
-    name: "WireTestingPackage",
+    name: "WireFoundation",
     platforms: [.iOS(.v15), .macOS(.v12)],
     products: [
-        .library(name: "WireTestingPackage", type: .dynamic, targets: ["WireTestingPkg"])
+        .library(name: "WireFoundation", targets: ["WireFoundation"]),
+        .library(name: "WireFoundationSupport", targets: ["WireFoundationSupport"]),
```

Comment:
Am I correct that this package (`WireFoundationSupport` ) is intended for use in test targets?

Reply:
Yes it is, like the other -Support targets we already have.